Mathlib/NumberTheory/NumberField/CanonicalEmbedding.lean

```diff
@@ -909,7 +1106,7 @@ theorem exists_ne_zero_mem_ideal_of_norm_le {B : ℝ}
   · rw [← Nat.cast_sum, sum_mult_eq, Nat.cast_pos]
     exact finrank_pos
 
-/- Necessary to prevent a timeout that ends at here. #10131 -/
+/- TODO: Remove!. Necessary to prevent a timeout that ends at here. #10131 -/
```

Comment:
There is a bunch of this `TODO` that I don't understand.

Reply:
(I also added these lines to a several functions that have the same problem).

Reply:
#10131 has been merged, so if there is still a problem we should at least modify the comment.

Reply:
Yes my refactor pushed a few unrelated few things over the limit.

Reply:
So what should I do about these messages. Change the text to say the problem is still there? Or try to fix it 😛

Reply:
Don't add any more todos, but leave the ones that are already there

Reply:
Well, I have similar problems in some functions added by this PR, so I need to do something about those.

Reply:
If you can fix them even better, but anyway I would remove (or say something more) the reference to #10131.

Reply:
Actually, it was easier to solve that I expected (although I am not sure why it works). The problem was that Lean timed out trying to synthesize `OfNat (FractionalIdeal (↥(𝓞 K))⁰ K)ˣ 1` but replacing `1` by `↑1` does the trick.